.github/workflows/ci-local.yml

```
@@ -0,0 +1,73 @@
+name: CI (Local Testing)
+permissions:
+  contents: read
+
+on:
+  push:
+    branches: [ main, develop, "feature/*" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)